process-link/src/main/kotlin/com/ritense/processlink/web/rest/dto/OpenProcessLinkResult.kt

```diff
@@ -16,7 +16,7 @@
 
 package com.ritense.processlink.web.rest.dto
 
-data class OpenTaskResult<T> (
+data class OpenProcessLinkResult<T> (
```

Comment:
Not quite agreeing with this rename.
I think `ProcessLinkTaskResult` fits better.

Reply:
well it's not only about opening a task anymore. it's to retrieve a form or form-flow as well

Reply:
`StartProcessLinkResult` then?

Reply:
It's used for 2 things:
1. Use a process link to open a task
2. Use a process link to open a start event
Both of those are activities, so I'd say `ProcessLinkActivityResult` makes most sense if we want to stick to the camunda naming.

Reply:
agreed. Done!